src/main/java/com/amihaiemil/zold/RtWallet.java

```diff
@@ -37,7 +37,7 @@
  * @author Mihai Andronache (amihaiemil@gmail.com)
  * @version $Id$
  * @since 0.0.1
- * @todo #19:30min Continue implementing Wallet operations.
+ * @todo #29:30min Continue implementing Wallet operations.
```

Comment:
@ammaratef45 , if it is clear what is left, could you please specify so (it is possible to make not only 1 todo, by the way)?

Reply:
What I meant was the creation of some puzzles. E.g.:
```
todo: #29: 30min Implement com.amihaiemil.zold.RtWallet#pay
todo: #29: 30min Implement com.amihaiemil.zold.RtWallet#find
todo: #29: 30min provide unit tests for com.amihaiemil.zold.RtWallet#getId and com.amihaiemil.zold.RtWallet#balance
```